packages/contracts-bedrock/src/dispute/SuperFaultDisputeGame.sol

```diff
@@ -302,6 +291,9 @@ contract SuperFaultDisputeGame is Clone, ISemver {
         // Should only happen if this is a new game type that hasn't been set up yet.
         if (root.raw() == bytes32(0)) revert AnchorRootNotFound();
 
+        // Prevent initializing right away with an invalid claim state that is used as convention
+        if (rootClaim().raw() == keccak256("invalid")) revert SuperFaultDisputeGameInvalidRootClaim();
```

Comment:
nit: Can we move `keccak256("invalid")` to be a constant?

Reply:
Also, can we add a test for this?

Reply:
https://github.com/ethereum-optimism/optimism/pull/14431/commits/669a1fed6ad5f5850d4d55a8f68849a87e988219 done